src/Plugin/Field/FieldFormatter/ErrandServicesFormatter.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\helfi_tpr\Plugin\Field\FieldFormatter;
+
+use Drupal\Component\Utility\SortArray;
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\Core\Field\Plugin\Field\FieldFormatter\EntityReferenceEntityFormatter;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\helfi_tpr\Entity\ChannelTypeCollection;
+use Drupal\helfi_tpr\Entity\Service;
+
+/**
+ * Field formatter to render errand service maps.
+ *
+ * @FieldFormatter(
+ *   id = "tpr_service_err_channel_list",
+ *   label = @Translation("TPR - Errand Service Channels List"),
+ *   field_types = {
+ *     "entity_reference"
+ *   }
+ * )
+ */
+
+class ErrandServicesFormatter extends EntityReferenceEntityFormatter {
```

Comment:
let's try to extend from formatter base as it seems we are not using now anything from EntityReferenceEntityFormatter